server/core/region.go

```diff
@@ -467,11 +467,29 @@ func (r *RegionInfo) GetBuckets() *metapb.Buckets {
 	return (*metapb.Buckets)(buckets)
 }
 
+// GetStorePeerApproximateSize returns the approximate size of the peer on the specified store.
+func (r *RegionInfo) GetStorePeerApproximateSize(storeID uint64) int64 {
+	peer := r.GetStorePeer(storeID)
+	if storeID != 0 && peer != nil && peer.IsWitness {
+		return 0
+	}
+	return r.approximateSize
+}
+
 // GetApproximateSize returns the approximate size of the region.
 func (r *RegionInfo) GetApproximateSize() int64 {
 	return r.approximateSize
 }
 
+// GetStorePeerApproximateKeys returns the approximate keys of the peer on the specified store.
```

Comment:
Where are these two functions used?

Reply:
Should delete these two functions, they come from https://github.com/tikv/pd/pull/5285.